int16  type
string time
string event_name
string level
string judge
sensor_msgs/Image img